frontend/src/components/ErrorFallback.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Alert, Row, Col } from "antd";
+import styled from "styled-components";
+
+const FallbackWrapper = styled.div`
+    margin: 200
+`;
+
+export const ErrorFallback = () => {
+    return (
+        <FallbackWrapper>
+            <Row justify="space-around" align="middle" >
```

Comment:
Also if we put a margin inside this component it can become difficult to reuse this inside of a specific list or modal since it adds a margin at top